vendor/ember/shims.js

```diff
@@ -0,0 +1,247 @@
+(function() {
+/* globals define, Ember, jQuery */
+
+  function processEmberShims() {
+    var shims = {
+      'ember': {
+        'default': Ember
+      },
+      'ember-application': {
+        'default': Ember.Application
+      },
+      'ember-array': {
+        'default': Ember.Array
+      },
+      'ember-array/mutable': {
+        'default': Ember.MutableArray
+      },
+      'ember-array/utils': {
+        'A':            Ember.A,
+        'isEmberArray': Ember.isArray,
+        'wrap':         Ember.makeArray
+      },
+      'ember-component': {
+        'default': Ember.Component
+      },
+      'ember-components/checkbox': {
+        'default': Ember.Checkbox
+      },
+      'ember-components/text-area': {
+        'default': Ember.TextArea
+      },
+      'ember-components/text-field': {
+        'default': Ember.TextField
+      },
+      'ember-controller': {
+        'default': Ember.Controller
+      },
+      'ember-controller/inject': {
+        'default': Ember.inject.controller
+      },
+      'ember-controller/proxy': {
+        'default': Ember.ArrayProxy
```

Comment:
This one felt surprising- `ArrayProxy` in the `ember-controller` namespace.

Re: https://github.com/ember-cli/ember-cli-shims/issues/60

I guess `SortableMixin` is in the same boat. Often used for controllers, but the coupling to the namespace seems odd (if we drop controllers, we will be forced to move/rename `ArrayProxy` and `SortableMixin`).